### DISCUSSION TITLE
Help with styles

### DISCUSSION THREAD

**Discussion by lariane-guerra** [gpbl/react-day-picker#2278] [Support]
Its possible to change in version v8? 

![image](https://github.com/user-attachments/assets/0842187d-9e00-4156-9edd-dfeb50475a64)

I need almost this (the second photo its from the playground react day pick):
<img width="343" alt="image" src="https://github.com/user-attachments/assets/1823972d-a6cb-497c-84cf-6508ae32276a">

**Comment by MrGhost23** [comment]
I'm still searching for the same thing. Right now, I don't have an answer. If you figure it out, please let me know.

**Comment by MrGhost23** [comment]
I found a solution!
works with modifiers

![image](https://github.com/user-attachments/assets/e61b7309-dc3f-4c2a-9253-471b818aa804)
